pkg/tfbridge/provider_test.go

```diff
@@ -960,6 +960,8 @@ func TestCheckConfig(t *testing.T) {
 	// Secrets: it is important to handle secrets correctly since the results of CheckConfig are stored in the
 	// state. If secrets are not marked as such, they are stored as plaintext in the state.
 	t.Run("preserve_program_secrets", func(t *testing.T) {
+		t.Skip()
```

Comment:
Why are these skipped?

Reply:
So these are skipped because they fail. We are basically counting on these test situations never happening because the engine takes care of the "program-originating" secrets for us. So it's not the bridged provider's responsibility anymore to track these but the engine's. Test can be deleted perhaps.

Reply:
Removed these.